src/resolvers/git.cr

```diff
@@ -39,13 +39,8 @@ module Shards
       if file_exists?(refs, SPEC_FILENAME)
         capture("git show #{refs}:#{SPEC_FILENAME}")
       else
```

Comment:
I suppose we shouldn't generate an empty `shard.yml` when it's missing anymore, but fail with an error instead.